### ISSUE TITLE
User should be able to type input if no selection is found

### ISSUE THREAD

**Issue by oslo-c** [oslo-c/norsk_lookup#1]
When the user tries the hotkey and no highlighted selection is found (either because they haven't selected anything or because accessibility features have not been made available), then the user should be able to type a word out into an input box and submit that via the api
